agent/consul/rpc.go

```diff
@@ -790,11 +783,6 @@ func (s *Server) raftApplyWithEncoder(
 		// In this case we didn't apply all chunks successfully, possibly due
 		// to a term change; resubmit
 		if resp == nil {
-			// This returns the error in the interface because the raft library
```

Comment:
Removed because this error unification already happened in a prior refactor?

Reply:
That is correct, yes. I missed removing the comment earlier.